engine/resources/nspawn.go

```diff
@@ -174,7 +172,6 @@ func (obj *NspawnRes) Watch() error {
 
 	defer close(busChan)
```

Comment:
I think this line should go right after the make() for it. Want to move it there in the same patch please?

Reply:
Is there any reason I shouldn't also move `bus.Close()` to right under `bus, err :=...`? I don't think the order matters in this case.

Reply:
Oh you definitely should AFAICT :)

I sometimes need to remind myself that present me is often a better programmer than past me, so as a result, I'm always cleaning up after old mistakes or missed mistakes, and should feel free to change anything that's wrong.